src/com/facebook/buck/swift/SwiftCompile.java

```diff
@@ -90,6 +92,7 @@ public String apply(String input) {
       };
 
   SwiftCompile(
+      CxxLink delegateLinkRule,
```

Comment:
This is definitely something I'll change. I originally wanted to copy XCode behavior of always creating dylib during `swiftmodule` compilation, this is just 1 to 1 behavior -> implementation mapping.